pycqed/measurement/waveform_control/pulsar.py

```diff
@@ -635,6 +658,12 @@ def s(val):
                     obj.set('sigouts_{}_range'.format(int(id[2])-1), 2*val)
             else:
                 s = None
+        elif par == 'amplitude_scaling' and id[-1] != 'm':
+            awg = int((int(id[2:]) - 1) / 2)
+            output = (int(id[2:]) - 1) - 2 * awg
+            def s(val):
+                obj.set(f'awgs_{awg}_outputs_{output}_amplitude', val)
+                log.debug(f'awgs_{awg}_outputs_{output}_amplitude: {val}')
```

Comment:
Can't really review the above lines because i've no idea what this function does and why these parameters needed special treatment.
I also cannot judge why only for id[-1] != 'm' and all the calculations here but i trust it's correct.